src/plugins/formvalid/formvalid-server-en.hbs

```diff
@@ -0,0 +1,127 @@
+</ul>
+
+<hr />
+
+<section>
+	<h2>Purpose</h2>
+	<p>This component merges Server and Client errors together.</p>
+</section>
+
+<section>
+	<h2>Example</h2>
+
+<p>After submitting your page, the server returned 2 server errors. Now if you try to resubmit your form with some client errors, the component will add these errors to the existing server errors.</p>
+<p>To test: Try clearing a field or enter 201 as the Beneficiary's age.</p>
+
+	<h2>
+		<span id="lblTitle" property="name">Demo Asp.Net Web Form</span>
+	</h2>
+
+	<div class="wb-frmvld">
+	<form id="frmMain" action="#" method="get">
+
+		<section tabindex="-1" class="alert alert-danger" id="errors-frmMain">
+			<h2>The form could not be submitted because 2 errors were found.</h2>
+			<ul>
+				<li><a href="#BeneficiaryName"><span class="prefix">Error&nbsp;1: </span>Beneficiary name - The name should begin with "Super" - (Correct and resubmit) </a></li>
+				<li><a href="#BeneficiaryAge"><span class="prefix">Error&nbsp;2: </span>Beneficiary age - Age should be greater than 64 for a Retirement benefit - (Correct and resubmit) </a></li>
+			</ul>
+		</section>
+
+		<noscript>
+			<!-- Fix Error cs0103 -->
+
+			<section class="alert alert-danger wb-server-error-clear" >
+			<h2>This is user control DisplayErrors (It appears when JavaScript is disabled)</h2>
+			<ul>
+			<li><a href="#BeneficiaryName"><span class='prefix'>Error&amp;nbsp;1: </span>Beneficiary name - The name should begin with "Super" - (Correct and resubmit)</a></li>
+			<li><a href="#BeneficiaryAge"><span class='prefix'>Error&amp;nbsp;2: </span>Beneficiary age - Age should be greater than 64 for a Retirement benefit - (Correct and resubmit)</a></li>
+			</ul>
+			</section>
+		</noscript>
+
+		<section class="panel panel-info">
+			<header class="panel-heading">
+				<h2 class="panel-title">
+					Section 1
+				</h2>
+			</header>
+
+			<div class="panel-body">
+				<div class="form-group">
+					<label class="css-star" for="TypeOfPrestation">
+						<abbr title="">
+							<span title="Enter the type of benefit desired" class="field-name" id="TypeOfPrestationLbl">Type of benefit</span>
+						</abbr>
+						<strong class="css-required">
+							(Required field)
+						</strong>
+					</label>
+					<div>
+						<select name="TypeOfPrestation" class="valid" id="TypeOfPrestation" aria-invalid="false" aria-required="true" required="required">
+							<option value="">Select a benefit</option>
+							<option value="ChildrenHelp">Child benefit (Age 1-18) </option>
+							<option value="Invalidity">Invalidity (Age 1-64)</option>
+							<option selected="selected" value="Retirement">Retirement (Age 65-200)</option>
+						</select>
+					</div>
+				</div>
+
+				<fieldset>
+				<legend>Beneficiary</legend>
+
+				<div class="form-group">
+					<label class="css-star" for="BeneficiaryName">
+						<abbr title="">
+							<span title="Enter the beneficiary name" class="field-name" id="BeneficiaryNameLbl">Beneficiary name</span>
```

Comment:
The ```id``` for this span is not required. Just the CSS class name ```field-name``` is sufficient

Reply:
removed